dev/io.openliberty.jakartaee10.internal_fat/fat/src/io/openliberty/jakartaee10/internal/tests/EE10FeatureCompatibilityTest.java

```diff
@@ -474,8 +430,8 @@ public void transportSecurityUsed() {
         ee10FeaturesThatEnableSsl.add("connectorsInboundSecurity-2.0");
```

Comment:
This is not an EE10 feature any longer so it can be removed.

Reply:
Ok, I will take that out while I'm here.

Reply:
Done